demo.py

```diff
@@ -68,7 +73,9 @@ def main():
   })
 
   # Sign and dump layout
-  layout.sign(alice_key)
+  print "Load alice private key to sign layout..."
+  alice_private = toto.util.prompt_import_rsa_key_from_file("alice")
+  layout.sign(alice_private)
```

Comment:
I see that you made this change, but nowhere in layout.py you changed to use the PEM (or ed25519) instead of the key dictionary... is this expected?